utils/rtty/files/rtty.config

```diff
@@ -0,0 +1,7 @@
+config rtty
+    option  interface   'lan'
+#   option  id          'My-Device'
+#   option  description 'Description of my device'
+    option  host        'your-server-host'
+    option  port        '5912'
+#   option  ssl         1
```

Comment:
Please `host`, `port`, or `interface` option so that no default instance will be added at all.  Thank you.

Reply:
Fixed.

Reply:
I don't see how this is fixed.  The default config now has all required options to add and run a instance.  Enlighten me if I am wrong.  A default off config means no rtty run at all.

Reply:
Now fixed.